extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -147,7 +151,23 @@ export class AmpAutocomplete extends AMP.BaseElement {
         error => {
           throw error;
         });
-    return json['items'] ? json['items'] : [];
+    return Promise.resolve(json['items'] ? json['items'] : []);
```

Comment:
Nit: it doesn't make any difference, but this syntax is often used in JavaScript `json['items'] || []`